imgui/core.pyx

```diff
@@ -2139,11 +2158,35 @@ cdef class _ImGuiViewport(object):
         return _cast_ImVec2_tuple(self._ptr.WorkPos)
     
     @property
-    def work_size(self):
```

Comment:
Did you remove this intentionally from the api?

Reply:
nop, typo. re added it in the latest commit